tensorboard/components/tf_scalar_dashboard/tf-scalar-chart.html

```diff
@@ -0,0 +1,316 @@
+        inner-h-t-m-l="[[_break(title)]]"></div>
+      <div class="content">
+        <content></content>
+      </div>
+    </div>
+    <style>
+      .title-container {
+        border-left: 4px solid;
+        padding-left: 5px;
+        margin-bottom: 10px;
+      }
+      .title {
+        font-size: 14px;
+        text-overflow: ellipsis;
+        overflow: hidden;
+      }
+      .content {
+        font-size: 12px;
+      }
+    </style>
+  </template>
+  <script>
+    Polymer({
+      is: "tf-card-heading",
+      properties: {
+        title: String,
+        color: {
+          type: String,
+          value: null,   // this property is optional
+        },
+        _borderColor: {
+          type: String,
+          computed: '_computeBorderColor(color)',
+          readOnly: true,
+        },
+      },
+      _computeBorderColor(color) {
+        return color || 'rgba(255, 255, 255, 0.0)';  // 100% transparent white
+      },
+      _break: function(input) {
+        return input.replace(/([\/_-])/g, "$1<wbr>")
+      },
+    });
+  </script>
+</dom-module>
+
+<!--
+  A component that fetches scalar data from the TensorBoard server and
+  renders it into a vz-line-chart.
+-->
+<dom-module id="tf-scalar-chart">
+  <template>
+    <!--
+      Uncomment these to test the behavior of tf-card-heading.
+      They render like this:
+          card expanded:  http://i.imgur.com/Q7lbvxk.png
+          not expanded:   http://i.imgur.com/M38yshU.png
+      (In the "not expanded" one, the vz-line-chart is way too small,
+      but that's just because the headings take up too much space. It
+      doesn't indicate any problems with the headings.)
+    -->
+    <!--
+    <tf-card-heading title="[[tag]]" color="red">
+      I'm a subtitle, for testing.
+    </tf-card-heading>
+    <tf-card-heading title="[[tag]]" color="blue">
+      Look, I'm like the image dashboard. It's also fine if my content becomes really exorbitantly wide. Nothing to see here.<br />
+      step <strong>500</strong><br />
+      <input type="range" style="width: 100%"></input>
+    </tf-card-heading>
+    -->
+    <tf-card-heading title="[[tag]]"></tf-card-heading>
+    <vz-line-chart
+      x-type="[[xType]]"
+      color-scale="[[_runsColorScale]]"
+      smoothing-enabled="[[smoothingEnabled]]"
+      smoothing-weight="[[smoothingWeight]]"
+      tooltip-sorting-method="[[tooltipSortingMethod]]"
+      ignore-y-outliers="[[ignoreYOutliers]]"
+    ></vz-line-chart>
+    <div style="display: flex; flex-direction: row;">
+      <paper-icon-button
+        selected$="[[expanded]]"
+        icon="fullscreen"
+        on-tap="_toggleExpanded"
+      ></paper-icon-button>
+      <paper-icon-button
+        selected$="[[_logScaleActive]]"
+        icon="line-weight"
+        on-tap="_toggleLogScale"
+        title="Toggle y-axis log scale"
+      ></paper-icon-button>
+      <span style="flex-grow: 1"></span>
+      <template is="dom-if" if="[[showDownloadLinks]]">
```

Comment:
(orthogonal) I wonder if we should just always show download links, and remove the extra bit of state from the UI.